### PR TITLE
Added Helm Chart Starter kit

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added helm chart starter

## Why
* To simplify chart generation

### CODE REVIEW

**Comment by osterman** [comment]
Please rename to `helm-chart-scaffolding`

**Comment by goruha** [comment]
Done

File: Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: Helm Chart
+name: helm-starter
```
Comment (osterman) [inline Chart.yaml]: Rename this

File: Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+version: 0.1.0
+appVersion: 0.0.1
+home: https://github.com/cloudposse/helm-starter
+icon: https://raw.githubusercontent.com/chartmuseum/chartmuseum/master/logo.png
```
Comment (osterman) [inline Chart.yaml]: Can you add this to the repo? https://cloudposse.com/wp-content/uploads/sites/29/2017/07/CloudPosse2-TRANSAPRENT.png as `logo.png`. Then link to the raw version of that.

File: README.md [hunk 3]
```diff
@@ -1 +1 @@
-# helm-starter
\ No newline at end of file
+# Helm Starter
```
Comment (osterman) [inline README.md]: Rename

File: Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: Helm Chart
+name: helm-chart-scaffolding
+version: 0.1.0
+appVersion: 0.0.1
+home: https://github.com/cloudposse/helm-starter
```
Comment (osterman) [inline Chart.yaml]: fix

File: Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+icon: https://raw.githubusercontent.com/chartmuseum/chartmuseum/master/logo.png
```
Comment (osterman) [inline Chart.yaml]: Can you add this to the repo? https://cloudposse.com/wp-content/uploads/sites/29/2017/07/CloudPosse2-TRANSAPRENT.png as logo.png. Then link to the raw version of that.

File: Chart.yaml (hunk 4 above)
Comment (goruha) [reply Chart.yaml]: Done

**Review by osterman — APPROVED** [review]

**Pull request merged**